ui/DesignSystem/Component/SegmentedControl.svelte

```diff
@@ -27,13 +27,16 @@
 <style>
   .segmented-control {
     display: flex;
+    width: fit-content;
+    border: 1px solid var(--color-foreground-level-3);
+    border-radius: 0.25rem;
   }
   .segmented-control:hover button.active:not(:hover) {
     background: none;
   }
   .segmented-control button {
     cursor: pointer;
-    padding: 8px 16px;
+    padding: 9px 16px;
```

Comment:
Should be rems?

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1683/commits/7ab714b2ad576552d8a6fcf77ee9f33f5b5641a3